uni/lib/controller/parsers/parser_library_occupation.dart

```diff
@@ -3,26 +3,29 @@ import 'dart:convert';
 import 'package:http/http.dart';
 import 'package:uni/model/entities/library_occupation.dart';
 
+// ignore_for_file: avoid_dynamic_calls
```

Comment:
If we do this too many times in `parser/`, maybe it's worth to ignore it in the directory after all

Reply:
I think it's not okay to ignore it on the directory, first we should try to make the code pass the linter and if it's not possible without damaging readability too much, we ignore that rule. This method could apply to another parts of the code that are not the parsers, so having the github annotation globally is not a bad idea

Reply:
Alright, let's use that rule